### PR TITLE
website - font and brand updates

### PR DESCRIPTION
**Pull request by rjspiker**
**NOTE:** The HashiCorp Marketing Team would like to launch this update in a coordinated fashion with font/brand updates to other HashiCorp product sites.  We'd like to get this approved, but hold off on merging at the moment.  Thanks.

---

This PR accomplishes the following:
- Add newly approved fonts (Gilmer, Metro Sans, Deja Vu) and remove deprecated fonts where no longer needed
- Global typography updates (font-size, line-height, etc), adding .g-type classes shared across sites
- A few spacing updates
- Color updates to match color scheme with other HashiCorp sites
- Button style updates

A preview can be seen at: https://consul-fonts.netlify.com

cc: @kylemac @mwickett

### CODE REVIEW

File: website/source/assets/stylesheets/_variables.scss [hunk 1]
```diff
@@ -1,65 +1,95 @@
...
 // Home
 $home-header-background-color: transparent;
-$home-header-link-color: $gray-darker;
+$home-header-link-color: $gray-1;
 $home-header-link-color-hover: $black;
 
 // Sidebar
 $sidebar-background-color: $white;
-$sidebar-font-size: $font-size - 2;
+$sidebar-font-size: $font-size-default - 2;
```
Comment (lkysow) [inline website/source/assets/stylesheets/_variables.scss]: The body font size is ending up 17px (see above) but the sidebar size is still at 14px. This looks weird imo. I like the body font size but I think the sidebar should be at 15 then. Not sure what's causing the mismatch.

File: website/source/assets/stylesheets/_inner.scss [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+
+  h5,
+  h6 {
+    @extend .g-type-display-6;
+  }
+
+  p,
+  li,
+  .alert {
+    font-size: 1.063rem;
```
Comment (lkysow) [inline website/source/assets/stylesheets/_inner.scss]: Ahh I see this is where the 1.063 is coming from. Why isn't this just set to the body size?

File: website/source/assets/stylesheets/_inner.scss [hunk 3]
```diff
@@ -16,14 +36,12 @@
   code,
   pre code,
   tt {
-    font-family: $font-family-monospace;
-    font-size: $font-size - 2;
-    line-height: 1.6;
+    line-height: 1.926em;
```
Comment (lkysow) [inline website/source/assets/stylesheets/_inner.scss]: I think the line-height for the code blocks is a bit too big now. It's computed to 26px whereas previously it was 20.
Now:
image


Before:
image

Maybe 1.626?

image

**Comment by rjspiker** [comment]
Thanks @lkysow.  Adjustments to the sidebar `font-size` as well as the `<code>` `line-height` have been made based on your feedback (across the other project sites as well 🎉 ).  Definitely appreciate the feedback.

**Comment by lkysow** [comment]
Looking better! 3 things I noticed this time:

Looks like `<code>` blocks have had their font colour changed.

Before:
image

After:
image

Links:
image


They're the same colour as links now which is confusing. Maybe make them the body font color and make the background-color a bit brighter so there's more contrast?

I think terraform.io gets this pretty perfect:
image
image


---
I'm guessing this isn't something we can change but the kerning with `/`'s looks weird. The space before the `/` is greater than after the `/`:
image


---
The font-size of the header navigation (e.g. Use Cases, Intro) is only 14px now which looks weird with the sidebar at 15. I think they should be the same size?

**Comment by rjspiker** [comment]
@lkysow made those adjustments to the nav and `<code>`.

As for the kerning, I don't think there's anything I can do for that.  I'll pass on your concern to the Marketing Design team so they are aware of it.

**Pull request merged**